lib/room.rb

```diff
@@ -0,0 +1,31 @@
+module Hotel
+  class Room
+    attr_reader :number, :room_cost, :reservations
+    
+    TOTAL_ROOMS = 20
+    
+    def initialize(number:, room_cost: 200.00, reservations: nil)
```

Comment:
It's safe to make an empty array a default value in Ruby.

```suggestion
    def initialize(number:, room_cost: 200.00, reservations: [])
```